### PR TITLE
Update READMEs about "literal variables"

### PR DESCRIPTION
**Pull request by mr0re1**
Support for indexing was added in #1377

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -1142,23 +1142,17 @@ The variable is referred to by the source, either vars for deploment variables
...
-Literal variables are not interpreted by `ghpc` directly, but rather embedded in the
-underlying module. Literal variables should only be used by those familiar
-with the underlying module technology (Terraform or Packer); no validation
-will be done before deployment to ensure that they are referencing
-something that exists.
-
-Literal variables are occasionally needed when referring to the data structure
-of the underlying module. For example, to refer to the subnetwork self link from
-a vpc module through terraform itself:
+Literal variables should only be used by those familiar
```
Comment (nick-stroud) [inline examples/README.md:1149]: You have removed some content that is still applicable and useful in my opinion such as:
> Literal variables are not interpreted by `ghpc` directly, but rather embedded in the underlying module.

> no validation will be done before deployment to ensure that they are referencing something that exists.

File: examples/README.md
Comment (nick-stroud) [inline examples/README.md:1]: Without context, it is not clear why this change is occuring. Please update the description. I would suggest adding a link to the PR that made the actual change in the description.

**Review by nick-stroud — CHANGES_REQUESTED** [review]

File: examples/README.md (hunk 1 above)
Comment (mr0re1) [reply examples/README.md:1149]: It's not true anymore, we do interpret them and validate that they are "referencing something that exists"

Comment (mr0re1) [reply examples/README.md:1149]: It's not true anymore, we do interpret them and validate that they are "referencing something that exists"

File: examples/README.md
Comment (mr0re1) [reply examples/README.md:1]: PR description was updated

**Review by nick-stroud — APPROVED** [review]
